src/components/view-add-profile/view-add-profile.template.html

```diff
@@ -0,0 +1,78 @@
+<md-tabs md-dynamic-height md-border-bottom md-center-tabs="true">
+
+	<md-tab label = "Company Profile">
+		<div layout="row" layout-xs="column" layout-margin>
+			<div flex>
+				<form class= "profile-form" name="CompanyInformationForm">
+					<md-card>
+						<md-card-content>
+							<md-toolbar md-scroll-shrink>
+								<div class="md-toolbar-tools">Logo and Cover</div>
+							</md-toolbar>
+							<md-input-container md-no-float class="md-block">
+								<md-icon><ng-md-icon icon="add_a_photo" ></ng-md-icon></md-icon>
+								<input ng-model="$ctrl.profile.photo" type="file" placeholder="Upload Photo">
+								<label>Logo</label>
+							</md-input-container>
+							
+							<md-input-container md-no-float class="md-block">
+								<md-icon><ng-md-icon icon="add_a_photo" ></ng-md-icon></md-icon>
+								<input ng-model="$ctrl.profile.photo" type="file" placeholder="Upload Photo">
+								<label>Cover Picture</label>
+							</md-input-container>
+
+							<md-input-container class="md-block">
+								<label>Youtube Video Link</label>
+								<textarea ng-model="$ctrl.profile.profileDescription" md-maxlength="150" rows="3" md-select-on-focus></textarea>
+							</md-input-container>
+
+						</md-card-content>
+					</md-card>
+				</form>
+			</div>
+			<div flex>
+				<form class= "profile-form" name="EducationForm">
+					<md-card>
+						<md-card-content>
+							<md-toolbar md-scroll-shrink>
+								<div class="md-toolbar-tools">Information</div>
+							</md-toolbar>
+							<!--<md-list flex>
+								<md-subheader class="md-no-sticky">Added Educations</md-subheader>
+								<md-list-item class="md-3-line" ng-repeat="refugeeEducation in $ctrl.educations">
+									<div class="secondary-button-padding" layout="column">
+										<h3>{{ refugeeEducation.schoolName }} - {{ refugeeEducation.degree }}</h3>
+										<p>({{ refugeeEducation.startDate | date:'mediumDate' }} - {{ refugeeEducation.endDate | date:'mediumDate' }})</p>
+										<md-button class="md-secondary button-delete" ng-click="$ctrl.removeEducation(refugeeEducation)">Delete</md-button>
+									</div>
+									<md-divider></md-divider>
+								</md-list-item>
+							</md-list>-->
```

Comment:
Please fix these education things. There should be nothing with "education" word. Otherwise it is very confusing for other developers